elide-datastore/elide-datastore-multiplex/src/main/java/com/yahoo/elide/datastores/multiplex/BridgeableTransaction.java

```diff
@@ -54,12 +55,27 @@
      * @param scope  Request scope
      * @return Loaded object from bridgeable store.
      */
-    Object bridgeableLoadObject(MultiplexTransaction muxTx,
+    default Object bridgeableLoadObject(MultiplexTransaction muxTx,
                                 Object parent,
                                 String relationName,
                                 Serializable lookupId,
                                 Optional<FilterExpression> filterExpression,
-                                RequestScope scope);
+                                RequestScope scope) {
+        EntityDictionary entityDictionary = scope.getDictionary();
+
+        if (lookupId == null) {
+            // according to contract, lookupId can be null
+            // in this case, we compute the lookupId
+            Object parentInstance = muxTx.loadObject(
+                    entityDictionary.getEntityClass(parent.getClass().getName()),
+                    entityDictionary.getId(parent),
+                    filterExpression, scope
+            );
+            lookupId = entityDictionary.getId(entityDictionary.getField(parentInstance, relationName));
```

Comment:
I understand what you are trying to do here, but I think the whole contract for store bridging is busted.  I don't know if there is a general way to get the ID from the parent.  It seems like the other store (the one we bridged from would have to supply it somehow).